GTG/gtk/editor/taskeditor.ui

```diff
@@ -4,403 +4,461 @@
   <object class="GtkWindow" id="TaskEditor">
     <property name="can_focus">False</property>
     <property name="title" translatable="yes">Task</property>
-    <property name="default_width">500</property>
-    <property name="default_height">400</property>
+    <property name="default_width">800</property>
+    <property name="default_height">500</property>
```

Comment:
Is it possible to keep the original default_width/default_height? Nice thing about the GTG task editor window was it  was quite small if you wanted.